src/ifp/src/InitFloorplan.tcl

```diff
@@ -263,4 +269,33 @@ proc microns_to_mfg_grid { microns } {
   }  
 }
 
+proc cut_rows {block placement_blockages} {
```

Comment:
Do you think there would be a benefit to move this to C++ so we could reuse some of the unit testing infrastructure there? It seems in general the project is moving away from TCL for top level logic.

Reply:
The unit testing is language agnostic.